pyproject.toml

```diff
@@ -1,8 +1,12 @@
 [build-system]
 # Minimum requirements for the build system to execute.
-requires = ["wheel", "setuptools>=30.3.0", "attrs>=17.1", "setuptools_scm"]
+requires = ["wheel", "setuptools>=30.3.0", "attrs>=17.1", "setuptools_scm[toml]>=3.4"]
 build-backend = "setuptools.build_meta"
 
+[tool.setuptools_scm]
+write_to = "src/pyhf/version.py"
+local_scheme = "no-local-version"
```

Comment:
@kratsg this is all fine, but can you add a link in your PR summary description that links to the relevant sections in the `setuptools_scm` docs to make the easier for people like me to understand what's happening in the future?

Reply:
They don't have official docs. Just the readme? https://github.com/pypa/setuptools_scm#pyprojecttoml-usage -- can't do much more than that.